rllib/connectors/env_to_module/prev_action_prev_reward.py

```diff
@@ -0,0 +1,134 @@
+from functools import partial
```

Comment:
Built-in prev action/reward connector.

Optionally move this into examples folder as RLlib does not use this automatically (user has to explicitly configure this connector via config.env_to_module_connector = lambda env: PrevRewardPrevActionEnvToModule(env=env, n_prev_actions=1, n_prev_rewards=10)).